assets/src/styles/components/hydrograph/_app.scss

```diff
@@ -109,16 +109,8 @@
     .tooltip-text-group {
         pointer-events: none;
         position: absolute;
-        background: rgba(255, 255, 255, 0.8);
-        margin: 0 0 0 40px;
-        font-size: 0.5em;
+        background: rgba(255, 255, 255, 0);
```

Comment:
@abriggs-usgs was removing the transparency on this intentional? It makes some things difficult to read - for instance on this site with temperature and the previous year selected: https://webvadevvs04.er.usgs.gov/wsgi/waterdataui/monitoring-location/01646500/

Reply:
Yes. it was to deal with the margin fading out the y-axis, but I do see what you mean. I made a new ticket to cover it:  https://internal.cida.usgs.gov/jira/browse/WDFN-287